test/jit/test_async.py

```diff
@@ -197,6 +197,23 @@ def wait_script(x1, x2, x3):
         self.assertEqual(y2, foo2(x1, x2))
         self.assertEqual(y3, foo3(x1, x2, x3))
 
+    def test_async_kwargs(self):
+        @torch.jit.script
+        def foo(x1, x2):
```

Comment:
Can you also test with a non-scripted function, and also while using `torch.jit.trace` ?

Reply:
I added different variants.